src/main/java/duke/printers/Cliui.java

```diff
@@ -0,0 +1,98 @@
+package duke.printers;
+
+import duke.definitions.CommandPacket;
+import duke.tasktypes.Task;
+
+import java.util.ArrayList;
+
+public class Cliui {
+    static String separator = "------------------------------------------";
+    static String indent = " ";
+    static String greet = " Hello! I'm duke.";
+    static String prompt = " What can I do for you?";
+    static String taskAdded = "Got it. I've added this task:";
+    static String taskDeleted = "Noted. I've removed this task:";
+    static String taskDone = "Nice! I've marked this task as done:";
+    static String goodbye = " Bye. Hope to see you again soon!";
+
+    public static void printSeparator() {
```

Comment:
From this line onwards - exemplary abstraction, with each small action defined into discrete methods that serve only a single purpose. Keep it up!